.github/workflows/ci.yaml

```diff
@@ -139,16 +138,16 @@ jobs:
           set -e
           cargo make --cwd llm build-ollama
           cd test
-          golem-cli app build -b ollama-debug
-          golem-cli app deploy -b ollama-debug
-          golem-cli worker new -e GOLEM_OLLAMA_BASE_URL=http://localhost:11434 test:llm/ollama-1
-          golem-cli worker invoke test:llm/ollama-1 test1
-          golem-cli worker invoke test:llm/ollama-1 test2
-          golem-cli worker invoke test:llm/ollama-1 test3
-          golem-cli worker invoke test:llm/ollama-1 test4
-          golem-cli worker invoke test:llm/ollama-1 test5
-          golem-cli worker invoke test:llm/ollama-1 test6
-          golem-cli worker invoke test:llm/ollama-1 test7
+          golem app build -b ollama-debug
+          golem app deploy -b ollama-debug
+          golem worker new -e GOLEM_OLLAMA_BASE_URL=http://localhost:11434 test:llm/ollama-1
+          golem worker invoke test:llm/ollama-1 test1 | grep -v "ERROR: " || exit 1
+          golem worker invoke test:llm/ollama-1 test2 | grep -v "ERROR: " || exit 1
```

Comment:
`|| exit 1` should not be necessary with `set -e`

Reply:
This is done now